rasa/core/constants.py

```diff
@@ -1,8 +1,14 @@
 DEFAULT_SERVER_PORT = 5005
 
-DEFAULT_SERVER_FORMAT = "{}://localhost:{}"
+DEFAULT_SERVER_INTERFACE = "0.0.0.0"
 
-DEFAULT_SERVER_URL = DEFAULT_SERVER_FORMAT.format("http", DEFAULT_SERVER_PORT)
+DEFAULT_SERVER_FORMAT = "{}://{}:{}"
```

Comment:
Can you check the code base for all usages of this and make we are using the correct number for variables whenever we call `.format`. (I think there is another usage in (`rasa/core/run.py`).

Reply:
> 
> 
> This is turning out to be trickier than expected!
> But I think we are close, just a couple more comments.

Same here. And I think I'm causing it by trying to retrofit. I think this approach may be better as it causes less changes to the other code

> 
> A tip, you can run `make lint` before pushing to make sure the commit will pass the code quality step.

Thanks. I'll see if I get that running on windows

> 
> Also, have you tried the new option with your usecase yet (i.e. the VPN) just to confirm it works as expected?

Yes, I tried it. It works! I even created a `systemd` configuration for it which I'll be happy to contrib